docs/workshops/interviews/technical/sample-questions.md

```diff
@@ -0,0 +1,39 @@
+---
+id: sample-questions
+title: Sample questions
+sidebar_label: Sample Questions
+---
+
+# Sample questions for technical interviews
```

Comment:
Could we add some instruction for TAs about how to run this? e.g. setup collavorative coding env,